internals/secrethub/audit.go

```diff
@@ -63,13 +90,34 @@ func (cmd *AuditCommand) run() error {
 		return err
 	}
 
-	tabWriter := tabwriter.NewWriter(cmd.io.Stdout(), 0, 4, 4, ' ', 0)
-	header := strings.Join(auditTable.header(), "\t") + "\n"
-	fmt.Fprint(tabWriter, header)
+	var formatter tableFormatter
+	if cmd.json {
+		formatter = newJSONFormatter(auditTable.header())
+	} else {
+		terminalWidth, err := cmd.terminalWidth(int(os.Stdout.Fd()))
+		if err != nil {
+			terminalWidth = defaultTerminalWidth
+		}
+		formatter = newColumnFormatter(terminalWidth, auditTable.columns())
+	}
+
+	paginatedWriter, err := cmd.newPaginatedWriter(os.Stdout)
+	if err == errPagerNotFound {
+		paginatedWriter = newFallbackPager(os.Stdout)
+	} else if err != nil {
+		return err
+	}
```

Comment:
This looks a bit weird to me. Why would the `newPaginatedWriter` not also include this logic for falling back to a `fallbackPager`?

Also, consider your naming: what is the difference between a `PaginatedWriter` and a `Pager`? They seem to do the same, so consider naming them consistently.

Reply:
I wrote this considering that there might be different ways of handling this error so, "whoever" asks for a pager and does not get one should deal with it however they want.

Reply:
And good point. I will name them consistently.